andela/neon-ah-backend
# Ft document api swagger 162414266

#### What does this PR do?
 Documents the Authors' Haven Application

#### Description of Task to be completed?
 Document API using swagger

#### How should this be manually tested?
 Check the documentation [link](http://localhost:3000/docs/#/) to confirm it is up

#### Any background context you want to provide?
 I have only a local URL for the documentation as the team is yet to get a Heroku or online link
 There a demo routes (curled from the readme file) and models in the documentation

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/n/projects/2229851/stories/162414266
